app/views/estimate_flow/housing_costs.html.slim

```diff
@@ -0,0 +1,10 @@
+- content_for :page_title
+  = t(".heading")
+- content_for :back do
+  = back_link(:housing_costs, @check, @back_buttons_invoke_browser_back_behaviour)
+.govuk-grid-column-two-thirds
+  = render "estimate_flow/forms/housing_costs", i18n_key: "housing_costs"
```

Comment:
I don't think there's actually any benefit having the form in its own partial here. We introduced that pattern where 2 views needed to render the same form (i.e. client and partner views that basically show the same thing). In this case, I think it would make life simpler if you deleted this line, and instead copied and pasted the contents of `estimate_flow/forms/_housing_costs.html.slim` into this file, indenting appropriately. Then everything we need would be in one place.

Reply:
I guess we'd do the same for the `app/views/estimate_flow/_mortgage_or_loan_payment.html.slim` partial. I'll amalgamate this one too.